art/attacks/inference/attribute_inference/black_box.py

```diff
@@ -106,12 +116,19 @@ def __init__(
         """
         super().__init__(estimator=estimator, attack_feature=attack_feature)
 
-        self._values: Optional[list] = None
-        self._attack_model_type = attack_model_type
-        self._attack_model = attack_model
+        self._values: list = []
```

Comment:
Empty lists should not be default values for arguments because if multiple instances of this class are created they will overwrite each others arguments.
```suggestion
        self._values: Optional[list] = None
```

Reply:
Please ignore.